src/Components/Banner/index.tsx

```diff
@@ -1,137 +1,136 @@
 import { media } from '@/Styles/theme';
 
-const useInterval: { (callback: () => void, interval: number): void } = (callback, interval) => {
-  const savedCallback = useRef<(() => void) | null>(null);
+export const createCarouselOptions = (handleSlideIdx: React.Dispatch<SetStateAction<number>>): Settings => {
+  return {
+    /** 배너의 contents 갯수로를 표시하는 dot의 유무 */
+    dots: false,
 
-  useEffect(() => {
-    savedCallback.current = callback;
-  });
+    /** 자동 스크롤 사용 유무 */
+    autoplay: true,
 
-  useEffect(() => {
-    function tick() {
-      if (savedCallback.current) {
-        savedCallback.current();
-      }
-    }
-    if (interval !== 10000) {
-      const id = setInterval(tick, interval);
-      return () => clearInterval(id);
-    }
-  }, [interval]);
+    /** 다음으로 넘어가는데 걸리는 시간(ms) */
+    autoplaySpeed: 5000,
+
+    /** 드래그 가능 여부 */
+    draggable: false,
+
+    /** 무한 반복 */
+    infinite: true,
+
+    /** 버튼으로 다음 이미지 보이는데 걸리는 시간(ms) */
+    speed: 400,
+
+    /** 배너에 보여질 이미지의 개수 */
+    slidesToShow: 1,
+
+    /** 앞뒤로 넘기는 arrow 유무 */
+    arrows: true,
+
+    /** 이전으로 이동하는 화살표 컴포넌트 */
+    prevArrow: <CustomArrowBtn type="PREV" />,
+
+    /** 다음으로 이동하는 화살표 컴포넌트 */
+    nextArrow: <CustomArrowBtn type="NEXT" />,
+
+    /** 슬라이더의 인덱스가 변경되기 직전에 호출되는 콜백함수 */
+    beforeChange: (_, next) => handleSlideIdx(next + 1),
+  };
 };
 
 /** 메인 페이지 배너 */
 const Banner = () => {
-  const outRef = useRef<HTMLDivElement>(null);
-  const slideRef = useRef<HTMLDivElement>(null);
-  const navigate = useNavigate();
-  const banners = [Banner2, Banner1, Banner2];
-  const [slideIdx, setSlideIdx] = useState(1);
-  const [slideInterval, setSlideInterval] = useState(5000);
-  useInterval(() => setSlideIdx((slideIdx) => slideIdx + 1), slideInterval);
-
-  if (slideIdx === 3) {
-    if (slideRef.current) slideRef.current.style.transition = '';
-
-    setSlideIdx(0);
-    setTimeout(() => {
-      if (slideRef.current) {
-        slideRef.current.style.transition = 'all 600ms ease-in-out';
-      }
-    }, 0);
-  }
-
-  useEffect(() => {
-    if (slideIdx === 0) {
-      setSlideInterval(0);
-    } else {
-      setSlideInterval(5000);
-    }
-  }, [slideIdx]);
+  const banners = [Banner1, Banner2];
+  const mobileBanners = [MobileBanner1, MobileBanner2];
+  const [slideIdx, setSlideIdx] = useState<number>(1);
+  const carouselOptions = createCarouselOptions(setSlideIdx);
 
   return (
-    <BannerWrapper ref={outRef}>
-      <BannerItemsWrapper ref={slideRef} $bannerLength={3} idx={slideIdx}>
+    <BannerBox>
+      <Carousel options={carouselOptions}>
         {banners.map((banner, idx) => (
-          <BannerItemWrapper key={idx} onClick={() => navigate(ROUTES.RECRUITMENT.RECRUITMENTS)}>
-            <img src={banner} />
-          </BannerItemWrapper>
+          <BannerItemList key={idx} to={NOTION_URL} target="_blank">
+            <BannerImage src={banner} srcSet={`${mobileBanners[idx]} 500w, ${banner}`} alt="banner" />
+          </BannerItemList>
```

Comment:
key로 idx만 사용할 경우 다른 요소와 중복될 수 있어서 앞에 요소명을 적는게 좋을 것 같아요!

```suggestion
          <BannerItemList key={`banner-${idx}`} to={NOTION_URL} target="_blank">
            <BannerImage src={banner} srcSet={`${mobileBanners[idx]} 500w, ${banner}`} alt="banner" />
          </BannerItemList>
```

Reply:
꼼꼼히 봐주셔서 감사합니다!! 수정사항 반영하겠습니다.